elements/pfe-band/src/pfe-band.scss

```diff
@@ -148,19 +148,23 @@ $LOCAL: band;
     --pfe-band--gridTemplateArea_mobile: "body";
     // Styles for the different combination of regions
     &[pfe-has-aside] {
-      --pfe-band--gridTemplateArea_mobile: "body" "aside";
-      --pfe-band--gridTemplateArea_desktop: "body aside";
+      --pfe-band--gridTemplateArea_mobile:
```

Comment:
These newlines are not necessary.  The layout issues are due to missing attributes, not to line breaks.  As long as grid templates use individual quotes around the layers, they don't need newlines.

Reply:
agreed, they don't change the resulting CSS. The line breaks are for readability, to view the regions as they would appear in a grid format.

Reply:
+1, I wasn't sure they would work with custom properties but they DO! 🎉